src/modules/scraper/entry-point/http/dto.ts

```diff
@@ -131,3 +131,10 @@ export class BackfillFeeBreakdownBody {
   @ApiProperty({ example: 0 })
   count?: number;
 }
+
+export class BackfillFilledDateBody {
+  @IsOptional()
+  @IsInt()
+  @ApiProperty({ example: 0 })
```

Comment:
Could we also include an example of this being undefined?

Reply:
Hmm, why do you want to add an example of this being undefined? 🤔